.github/workflows/continuous-integration.yml

```diff
@@ -1,12 +1,18 @@
 
 name: "Continuous Integration"
 
-on: ["pull_request", "push"]
+on:
+  pull_request:
+    branches: '**'
+  push:
+    branches: '**'
```

Comment:
This is why when somebody opens a PR from the upstream repo, we get all jobs run twice. Why don't we restrict these to only the release branches?

Reply:
impossible - I already spent a lot of time figuring this out

Reply:
What do you mean? I pushed something, it seems to work…

Reply:
> the branch here is the target branch, right?

It should be. At least it is on Travis and AppVeyor where this does work.

Reply:
> Why don't we restrict these to only the release branches?

pull request - must be for all branches, otherwise CI results will not be displayed in PR if from another repo
push - we must not limit, otherwise it CI will not run in user's repo

so as long as GH Actions are free, this should not be an issue if it is run twice (once for push, once for PR)

Reply:
> so as long as GH Actions are free, this should not be an issue if it is run twice (once for push, once for PR)

It has been the exact issue in the past with Travis. Such builds would take twice longer for no reason.